doc/rest.md

```diff
@@ -6,11 +6,27 @@ import {Rest} from 'spoonx/aurelia-api';
 
 ----------
 
-## .request(method, path[, body][, options])
+## Properties
```

Comment:
These changes also require removing spoonx/ from the examples. I'll move it, if you remove docs and add a readme that points here? :)